### PR TITLE
Update medasdigital chain-id and osmosis denom and inc channel

### PR DESCRIPTION
**Pull request by oxygene76**
## Description

Updates medasdigital on osmosis
Updated chain-id
Updated on bc denom for Medas
Updated ibc channel medasdigital-> osmosis

Pool for new medasdigital ibc denom will be created tomorrow and updated in assetlist.

### CODE REVIEW

**Comment by oxygene76** [comment]
The new Pools have been created

- gamm/pool/2333
- gamm/pool/2334

File: osmosis-1/osmosis-1.chainlist.json [hunk 1]
```diff
@@ -4986,8 +4986,8 @@
       "chain_name": "medasdigital",
       "status": "live",
       "network_type": "mainnet",
-      "pretty_name": "Medas Digital Network",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.chainlist.json:4989]: Please don't update this chainlist file directly

Comment (oxygene76) [reply osmosis-1/osmosis-1.chainlist.json:4989]: Ok, sorry seeing it, only osmosis.zone_assets.json.. The Pool info is now automatically pulled? Last time I had to edit some file and enter the initial osmo pool..

Comment (oxygene76) [reply osmosis-1/osmosis-1.chainlist.json:4989]: Ok, should I do a new PR without the chainlist or can you ignore the chainlist for the merge?

Comment (JeremyParish69) [reply osmosis-1/osmosis-1.chainlist.json:4989]: Just undo the changes to that file in your branch. We can still keep this PR

Comment (oxygene76) [reply osmosis-1/osmosis-1.chainlist.json]: OK, thx.. I reverted the changes.. so for osmosis only the change of the ibc channel and the features seems to be required..

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -1382,7 +1382,7 @@
     {
       "chain_name": "medasdigital",
       "base_denom": "umedas",
-      "path": "transfer/channel-519/umedas",
+      "path": "transfer/channel-87042/umedas",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:1385]: This updated IBC connection has to be registered

Comment (oxygene76) [reply osmosis-1/osmosis.zone_assets.json:1385]: Sorry saw it and forgot the pull request of the changed files, did it instantly..

**Pull request merged**